04/mobile.css

```diff
@@ -0,0 +1,137 @@
+.navigation {
+    height: 15vh;
+    margin: 0 auto;
+    position: relative;
+}
+
+.nav {
+    display: none;
+    width: 100vw;
+}
+
+label {
+    border-top: 5px solid white;
+    width: 20px;
+    height: 20px;
+    position: absolute;
+    width: 8%;
+    top: 40%;
+    left: 46%;
+}
+
+label:before, label:after {
+    content: '';
+    min-width: 100%;
+    position: absolute;
+    top: 50%;
+    left: 0;
+    border-top: 5px solid white;
+}
+
+label:after {
+    transform: translateY(13px);
+}
+
+input {
+    display: none;
+}
+
+input:checked + .nav {
+    display: block;
+    position: absolute;
+    top: 100%;
+}
+
+.nav__item {
+    background-color: var(--color-gamma);
+    width: 33.3vw;
+    height: 10vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    font-size: 1.4rem;
+    position: relative;
+}
+
+.nav__item:hover {
+    background-color: #f6f6f6;
+}
+
+.nav__item:hover > a {
+    color: var(--color-alfa);
+}
+
+.nav__dropdown {
+    position: absolute;
+    left: 100%;
+    background-color: var(--color-gamma);
+    width: 33.3vw;
+    height: 100%;
```

Comment:
Coś chyba za dużo tych procentów używasz :P
Zobacz, % to zasze element zależy od rodzica. Tutaj masz li, który ma `10vh` czyli tyle samo będzie miał `ul`.
Ale!!! Ten `ul` ma 2 elementy `li` więc one średnio będą chciały się tam zmieścić...  bo są 2.

PS. Dodaj jeszcze `top: 0` do wszystko będzie równe ;)